### ISSUE TITLE
#1339613828------------- Links must have discernible text

### ISSUE THREAD

**Issue by khyatit13** [khyatit13/Coding-practice#497]
<strong>Source:</strong>`<a id="edit-button-9f75698c7e8a420da643577730bb6eaf" href="/layers/9f75698c7e8a420da643577730bb6eaf/edit" aria-describedby="calcite-tooltip-3522afab-cd80-803c-98b9-19838c4a4814">`
<strong>Element Location: </strong>#edit-button-9f75698c7e8a420da643577730bb6eaf
<strong>Impact: </strong>serious
<strong>failureSummary:</strong> Fix all of the following:
  Element is in tab order and does not have accessible text

Fix any of the following:
  Element does not have text that is visible to screen readers
  aria-label attribute does not exist or is empty
  aria-labelledby attribute does not exist, references elements that do not exist or references elements that are empty
  Element has no title attributeTags: [Ljava.lang.String;@aa794a3
Screenshot:`C:\Users\khy12817\Desktop\Test.PNG`
